lib/teaspoon/configuration.rb

```diff
@@ -25,13 +25,14 @@ class Configuration
 
     # console runner specific
 
-    cattr_accessor :driver, :driver_options, :driver_timeout, :server, :server_port, :server_timeout, :fail_fast,
+    cattr_accessor :driver, :driver_options, :driver_timeout, :server, :server_host, :server_port, :server_timeout, :fail_fast,
```

Comment:
Sorry @houndci, but splitting this line just to please style will harm the git history of the line below. I prefer the keep the git history as clean as possible so changes are easier to find when tracing a bug.